frontend/ui/src/pages/Channels/ChannelsMainPage/Sources/FacebookSource.tsx

```diff
@@ -16,11 +23,30 @@ const FacebookSource = (props: FacebookSourceProps) => {
         text="Connect multiple Facebook pages"
         image={<FacebookLogo />}
         displayButton={!channels.length}
+        id={ChannelSource.facebook}
+        onAddChannelClick={() => props.showDialogAction(ChannelSource.facebook)}
       />
 
-      <SourceInfo source="facebook" channels={channels} connected="CONNECTED" isConnected="connected" />
+      <SourceInfo
+        source="facebook"
+        channels={channels}
+        connected="CONNECTED"
+        isConnected="connected"
```

Comment:
Something about these props doesn't seem right if they reflect the same state, right? Is there a way that we can merge them?

Reply:
the prop `connected` has a bad naming, since I think this is the string that we show when we have more than X items in the components. Fixing this would be nice, maybe we don't even new a prop for this and can be inside the component itself.